integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/message/DBMessagePersister.java

```diff
@@ -0,0 +1,49 @@
+package no.difi.meldingsutveksling.nextmove.message;
+
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.nextmove.ConversationResource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import javax.transaction.Transactional;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+@ConditionalOnProperty(name = "difi.move.nextmove.useDbPersistence", havingValue = "true")
+public class DBMessagePersister implements MessagePersister {
+
+    private IntegrasjonspunktProperties props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "props" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)